src/app/hooks/useElements.ts

```diff
@@ -17,12 +17,28 @@ interface Options {
     page: Page;
     sort: Sort;
     filter: Filter;
+    address?: string;
```

Comment:
Is there a specific reason you explode the object in place of just adding `search: SearchParameters`?